### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by taliaga**
Let's the first proofread begin.

You can preview the docs at [https://smartsdk.github.io/ngsi-timeseries-api/](https://smartsdk.github.io/ngsi-timeseries-api/)

Ignore the broken links in the README.md, those I'll fix later.

Oh, and don't hesitate to use the online editor for minor fixes hehe.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,28 @@
...
-
-## Important things to keep in mind
-
-- Quantumleap needs to know where CrateDB is deployed. You can set this with the variable ```CRATE_HOST```. e.g ```CRATE_HOST=http://0.0.0.0:4200``` 
-
-- Notifications must come in complete [NGSI JSON Entity Representation](http://docs.orioncontextbroker.apiary.io/#introduction/specification/json-attribute-representation) form. Other forms, such as simplified entity representation, are not supported by QL because they lack information on attribute types, required by QL to make proper translations.
+Quantumleap is an adapter aimed to bring [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) Historical Data on top of TimeSeries Databases.
```
Comment (chicco785) [inline README.md]: QuantumLeap is an API that support the storage of NGSI [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) data into a [time series database](https://en.wikipedia.org/wiki/Time_series_database).

File: README.md [hunk 2]
```diff
@@ -1,95 +1,28 @@
...
 
+Want to know more? Head to the docs by clicking [here](https://smartsdk.github.io/ngsi-timeseries-api/).
```
Comment (chicco785) [inline README.md]: Want to know more? Head to the [docs](https://smartsdk.github.io/ngsi-timeseries-api/).

File: doc/manuals/admin/crate.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Crate
+
+[**Crate**](https://crate.io) is Quantumleap's default backend where NGSI data will be persisted. Until Quantumleap implements its querying API, you can interact directly with Crate to query all the data QL has saved from the received notifications.
```
Comment (chicco785) [inline doc/manuals/admin/crate.md]: [**Crate**](https://crate.io) is QuantumLeap's default backend where NGSI data will be persisted. Until QuantumLeap implements its querying API, you can interact directly with Crate to query all the data QuantumLeap has stored from the received notifications.

File: doc/manuals/admin/crate.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+If you followed the [Installation Guide](./index.md), you have already Crate running in a Docker container, ready to be used.
```
Comment (chicco785) [inline doc/manuals/admin/crate.md]: If you followed the [Installation Guide](./index.md), you have a ready-to-use Crate instance running in a Docker container.

File: doc/manuals/admin/crate.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+The easiest way to interact with it is using its admin interface, as documented [here](https://crate.io/docs/crate/guide/getting_started/connect/admin_ui.html). If not, you can use its [HTTP api](https://crate.io/docs/crate/guide/getting_started/connect/rest.html), or any of its supported drivers for different programming languages.
```
Comment (chicco785) [inline doc/manuals/admin/crate.md]: The easiest way to interact with Crate is using its admin interface, as documented [here](https://crate.io/docs/crate/guide/getting_started/connect/admin_ui.html). Alternatively, you can use its [HTTP API](https://crate.io/docs/crate/guide/getting_started/connect/rest.html), or any of the [supported clients](https://crate.io/docs/crate/guide/getting_started/clients/index.html).

File: doc/manuals/admin/crate.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+You can learn more about Crate by reading the docs at [https://crate.io/docs/crate/reference/](https://crate.io/docs/crate/reference/).
```
Comment (chicco785) [inline doc/manuals/admin/crate.md]: You can learn more about Crate by reading its [documentation](https://crate.io/docs/).

File: doc/manuals/admin/index.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+# Installing
+
+At the moment, the only supported way of using Quantumleap is through Docker, so there's no actual "installation" of Quantumleap.
```
Comment (chicco785) [inline doc/manuals/admin/index.md]: At the moment, the only actively supported distribution of QuantumLeap is Docker. Though, you can build and install it from sources, but no guidance is provided for such installation at the moment.

File: doc/manuals/admin/index.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+If you need to install Docker, refer to [Docker Installation](https://docs.docker.com/engine/installation/).
+
+The Quantumleap Docker Image is hosted at [https://hub.docker.com/r/smartsdk/quantumleap/](https://hub.docker.com/r/smartsdk/quantumleap/).
+
+Now, depending on your scenario, you might have different needs. See from the sections below which fits yours.
+
+
+## All-in-one
```
Comment (chicco785) [inline doc/manuals/admin/index.md]: Based on the code i provided for crate, I would provide also how to run in it a Swarm cluster.

File: doc/manuals/index.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+# Quantumleap
+
+Quantumleap is an adapter aimed to bring [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) Historical Data on top of TimeSeries Databases.
```
Comment (chicco785) [inline doc/manuals/index.md]: QuantumLeap is an API that support the storage of NGSI [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) data into a [time series database](https://en.wikipedia.org/wiki/Time_series_database).

File: doc/manuals/user/contributing.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
+## Contributions
```
Comment (chicco785) [inline doc/manuals/user/contributing.md]: link to the guide from my branch or import it?

File: doc/manuals/user/index.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+        "subject": {
+            "entities": [
+            {
+                "idPattern": ".*",
+                "type": "Room"
+            }
+            ],
+            "condition": {
+                "attrs": [  # Here the list of attrs whose change will trigger notifications. Empty array means track any change.
+                "temperature",
```
Comment (netzahdzc) [inline doc/manuals/user/index.md]: A minimal typo: "temperature" shouldn't have comma in this example

File: doc/manuals/user/index.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+                ]
+            }
+        },
+        "notification": {
+            "http": {
+                "url": http://quantumleap:8668/notify
+            },
+            "attrs": [  # Here the list of attrs to be persisted in time. Empty array means save all entity attributes.
+            "temperature",
```
Comment (netzahdzc) [inline doc/manuals/user/index.md]: Item without comma in this example :)

File: doc/manuals/admin/index.md (hunk 8 above)
Comment (taliaga) [reply doc/manuals/admin/index.md]: I think it'd rather add that option in a new PR. That deployment will require a bit more care in the docs :)

File: doc/manuals/user/index.md (hunk 11 above)
Comment (taliaga) [reply doc/manuals/user/index.md]: Thanks @netzahdzc!

File: doc/manuals/admin/index.md (hunk 8 above)
Comment (chicco785) [reply doc/manuals/admin/index.md]: Just put a placeholder with some coming soon text

File: doc/manuals/admin/crate.md (hunk 4 above)
Comment (chicco785) [reply doc/manuals/admin/crate.md]: it looks like you didn't implement my text edits.

**Comment by taliaga** [comment]
Something else maybe?

**Comment by chicco785** [comment]
I think there are still few missing links, that's all 👍

**Comment by taliaga** [comment]
@chicco785 

> I think there are still few missing links

Just let me know which are those and I can include them

File: README.md [hunk 13]
```diff
@@ -1,95 +1,28 @@
...
-- Notifications must come in complete [NGSI JSON Entity Representation](http://docs.orioncontextbroker.apiary.io/#introduction/specification/json-attribute-representation) form. Other forms, such as simplified entity representation, are not supported by QL because they lack information on attribute types, required by QL to make proper translations.
+QuantumLeap is an API that support the storage of NGSI [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) data into a [time series database](https://en.wikipedia.org/wiki/Time_series_database).
 
+Want to know more? Head to the [docs](https://smartsdk.github.io/ngsi-timeseries-api/).
 
-## Development Setup and Structure
+## Installation and Configuration
 
-The development is mostly python3 based for now, and really in the early stages so things will change for sure. For now, you can start with:
+Refer to the [Installation Guide]().
```
Comment (chicco785) [inline README.md]: link to guide is missing

File: README.md [hunk 14]
```diff
@@ -1,95 +1,28 @@
...
 
-    git clone https://github.com/smartsdk/ngsi-timeseries-api.git
-    cd ngsi-timeseries-api
-    python3 -m venv env
-    pip install -r requirements.txt
+## Getting started
 
-    # if you want to test everything locally, you'll need to...
-    source setup_dev_env.sh
+Refer to the [User Manual]().
```
Comment (chicco785) [inline README.md]: link to user manual is missing

File: README.md [hunk 15]
```diff
@@ -1,95 +1,28 @@
...
 
-Pytest is used as the testing framework, but since most of QL's functionality is integration of components, you'll find ```docker-compose.yml``` files in the test folders to be run as a setup for tests. If you see ```.travis.yml``` file you'll see how they are running today, but probably at some point it's worth exploring pytest-docker plugins.
+## Troubleshooting
 
-The ```requirements.txt``` still needs to be split between testing and production, that's why the docker image is massive for now.
+Refer to the [Troubleshooting]() section.
```
Comment (chicco785) [inline README.md]: Troubleshooting link is missing

File: README.md [hunk 16]
```diff
@@ -1,95 +1,28 @@
...
+## Contributing
 
-- **ngsi-timeseries-api**
-    - **client**: holds a simple Orion Context Broker client to ease integration testing. To be moved out of here at some point.
-    - **experiments**: sandbox for quick manual tests to try some stuff and derive new test cases.
-    - **python-flask** : will hold the implementation of the swagger-defined API controllers.
-    - **reporter**: this module is acting as the receiver of the notifications, who "parses/validates" them before being handled to the translators.
-    - **translators**: specific translators for timeseries databases.
-    - **utils**: common shared stuff looking for a better place to live in.
+Refer to the [Contributing]() section.
```
Comment (chicco785) [inline README.md]: Contributing link is missing

**Comment by taliaga** [comment]
Ahh, the ones in the main README! I left them to the end because they depend on how the docs end up structured. Thanks for the reminder though.
Pushed!

**Comment by taliaga** [comment]
I'm only missing in the contributing section the steps adjusted to the fork-based approach

**Comment by taliaga** [comment]
Actually, that was done. The doc was outdated :) so, the missing piece is actually the auto-update (publish) of docs to be added to travis workflow

**Pull request merged**